### DISCUSSION TITLE
How to insert image in the note panel ?

### DISCUSSION THREAD

**Discussion by djvasile** [freeplane/freeplane#1700] [General]
Hi, please advise me if the functionality to insert/paste images only in the note panel, not in the map exists.

I have installed [insertInlineImage](https://github.com/freeplane/addons/tree/master/insertInlineImage) however when i click to open folder it crashes.

Thank you very much, i want to add that i am a beginner at FreePlane, any help/advice is welcome

**Comment by euu2021** [comment]
InsertInlineImage is working fine here on Windows 11. I use this version v1.1: https://docs.freeplane.org/getting-started/Add-ons_(install).html#-insert-inline-image

**Comment by djvasile** [comment]
I had version 1.1 and it didnt work, but i updated from freeplane to 1.3 and it works now, howver i wish i could resize the image inside the note panel in order to make the text bigger to read from it, or perhaps to be able to open and close images
